### PR TITLE
New structure for intakt

### PR DESCRIPTION
**Pull request by GitThisJoel**
This PR is the first step towards intakt 2.0, I have replaced the old parsers with placeholder files, cleaned up some old files and reworked the structure of the program. There is still a lot of work to be done, and the old version is preserved on a separate branch called `intakt1.0`.

### CODE REVIEW

File: main/assets/masters.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "name": "Victor Winkelmann",
+        "stilid": "vi6253wi-s"
+    },
+    "dchip": {
+        "proper_name": "D-chip",
+        "name": "Klara Tjernström",
+        "stilid": "kl4257tj-s"
+    },
+    "medalj": {
+        "proper_name": "Medaljelele",
```
Comment (ax6160sv-s) [inline main/assets/masters.json:33]: Medaljelelekommitén

File: main/parse_zettle.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Used to generate the JSON-object from the Zettle API respnose.
+
+from datetime import datetime
+import argparse
+
+from get_zettle_purchases import get_sales
```
Comment (ax6160sv-s) [inline main/parse_zettle.py:6]: kanske klarare om man bara importerar get_zettle_purchases

File: main/get_zettle_purchases.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+from datetime import datetime
+from datetime import timedelta
+
+access_file = "credentials/access.json"
+with open(access_file) as f:
+    access_cred = json.load(f)
+    f.close()
+
+client_id = access_cred["client_id"]
+client_secret = access_cred["client_secret"]
```
Comment (ax6160sv-s) [inline main/get_zettle_purchases.py:14]: Är de här nödvändiga? Kan t.o.m. vara klarare utan

**Review by ax6160sv-s — COMMENTED** [review]
småsaker bara

File: main/get_zettle_purchases.py (hunk 3 above)
Comment (GitThisJoel) [reply main/get_zettle_purchases.py:14]: Tänker att vi tar bort det längre fram

File: main/assets/masters.json (hunk 1 above)
Comment (GitThisJoel) [reply main/assets/masters.json]: fixed

**Pull request merged**